spec/requests/placements/schools/mentors/add_mentor_spec.rb

```diff
@@ -2,14 +2,15 @@
 
 RSpec.describe "'Add mentor' journey", service: :placements, type: :request do
   let(:current_user) { create(:placements_user, schools: [school]) }
-  let(:start_path) { new_add_mentor_placements_school_mentors_path(school_id:) }
+  let!(:state_key) { SecureRandom.uuid }
```

Comment:
Just something about the fact that we pre-determine the state key here.. it feels weird, because presumably that means we never exercise the code that generates a _new_ state key from the controller. Right?

(Except for hitting it in the system specs. Where we sort of don't pay much attention to it. Maybe that's okay?)

Reply:
I added this originally because we can't match the path without the state key for the expectation, how do you feel about stubbing the value instead?

```ruby
  let(:state_key) { SecureRandom.uuid }

  before do
    sign_in_as current_user
    allow(SecureRandom).to receive(:uuid).and_return(state_key)
  end
```